AmazonChimeSDKDemo/AmazonChimeSDKDemo/CaptionCell.swift

```diff
@@ -7,8 +7,10 @@
 //
 
 import UIKit
+import AmazonChimeSDK
 
 let captionCellReuseIdentifier = "captionCell"
+let threshold = 0.3
```

Comment:
`threshold` for what? please use more descriptive naming, such as `lowItemConfidenceThreshold`

Reply:
will be updated.